react-multiple-pages/my-app/src/pages/Details.tsx

```diff
@@ -0,0 +1,33 @@
+import { Link, useParams } from 'react-router-dom';
+import { items } from '../lib/read';
+
+export function Details() {
+  const { itemId } = useParams();
+  const currentItem = items.find(
```

Comment:
Once again, you're not allowed to access `items` directly. You should read the data for a single item using the `readItem` function, passing it the `itemId` you get from `useParams`.

And, once again, the starter files do this so I'm not sure why you felt you needed to change it.